jobserver/management/commands/inflate_release_files.py

```diff
@@ -0,0 +1,129 @@
+import csv
+import json
+
+from django.core.management.base import BaseCommand
+
+from ...models import ReleaseFile
+
+
+def make_csv(release_file):
+    with release_file.absolute_path().open("w") as f:
+        columns = ["id", "created_at", "workspace_name", "release_id"]
+        writer = csv.DictWriter(f, columns)
+
+        writer.writeheader()
+        for _ in range(100):
```

Comment:
suggestion: The `100` that appears a few times feels a little magic. I assume it's somewhat arbitrary, but might be helpful to explain it.